packages/components/drawer/src/drawer.vue

```diff
@@ -51,6 +51,9 @@
               <slot></slot>
             </section>
           </template>
+          <div v-if="$slots.footer" class="el-drawer__footer">
```

Comment:
Maybe it's better to use the `footer` tag here? Same as above tag.

Reply:
is that be different from div?
I just followed the writing of other components
![image](https://user-images.githubusercontent.com/21104054/149689807-3c74a5a6-8d12-492f-abd2-ea7449e33071.png)

if there has some optimization by change `div` to `footer`, maybe we can make a new pr